README.rst

```diff
@@ -453,6 +457,18 @@ How to Cite
 
 If you utilize libCEED please cite::
 
+   @article{libceed-joss-paper,
+     author       = {Jed Brown and Ahmad Abdelfattah and Valeria Barra and Natalie Beams and Jean Sylvain Camier and Veselin Dobrev and Yohann Dudouit and Leila Ghaffari and Tzanio Kolev and David Medina and Will Pazner and Thilina Ratnayaka and Jeremy Thompson and Stan Tomov},
+     title        = {libCEED}: Fast algebra for high-order element-based discretizations},
```

Comment:
```suggestion
     title        = {{libCEED}: Fast algebra for high-order element-based discretizations},
```
Braces need balancing. I'll fix this, rebase onto `release` and re-target the PR to `release`.

Reply:
Sorry, that was a slip! I need to fix my local docs build to preview before pushing